gmso/tests/base_test.py

```diff
@@ -50,13 +50,13 @@ def _topology(sites=1):
             top = Topology()
             top.box = Box(lengths=[1, 1, 1])
             H = Hydrogen
-            site1 = Site(name='site1',
-                         element=H,
-                         atom_type=AtomType(name="at1",
-                                            mass=H.mass),
-                         )
             for i in range(sites):
```

Comment:
if you set `site=3`, would this work?

or is it not expected?

Reply:
There are better pytest fixtures that exist now, so I'm now using the `typed_ar_system` for these tests.  And then I plan to remove the `topology_site` fixture in a separate PR.